Explorer/Assets/Scripts/ECS/Unity/GLTFContainer/Asset/Systems/CreateGltfAssetFromAssetBundleSystem.cs

```diff
@@ -71,11 +66,6 @@ private void ConvertFromAssetBundle(in Entity entity, ref GetGltfContainerAssetI
                 return;
             }
 
-            if (!instantiationThrottler.Acquire(assetBundleData.GameObjectNodes.Count))
```

Comment:
Do you remember we were talking about removing the list and replacing it with a single object to exclude bias and not break each iteration by budgeting?